packages/flutter/lib/src/services/text_input.dart

```diff
@@ -1196,8 +1196,10 @@ abstract class DeltaTextInputClient extends TextInputClient {
   /// to the client's editing state. A change is any mutation to the raw text
   /// value, or any updates to the selection and/or composing region.
   ///
-  /// Here is an example of what implementation of this method could look like:
   /// {@tool snippet}
+  /// This example shows what an implementation of this method could look like.
+  ///
+  /// ```dart
```

Comment:
Good catch! Looks like the test is failing regarding some variables not being defined. I think something like this would make sense, removing `_previousValue`, what do you think? 
```
  /// TextEditingValue _localValue;
  /// @override
  /// void updateEditingValueWithDeltas(List<TextEditingDelta> textEditingDeltas) {
  ///   TextEditingValue newValue = _localValue;
  ///   for (final TextEditingDelta delta in textEditingDeltas) {
  ///     newValue = delta.apply(newValue);
  ///   }
  ///   _localValue = newValue;
  /// }
  /// ```
```

Reply:
Ah that's right.  That looks good to me, I'll try it.   Thanks!